### DISCUSSION TITLE
markdown file no longer output for multi_agent research?

### DISCUSSION THREAD

**Discussion by sdisaacson** [assafelovic/gpt-researcher#815] [General]
Hello!
I'm only getting .docx and .pdf files output from runs of multi_agent even though I've specified I want md in my task.json file.
is this missing by mistake or design?

My hope is for a return!

**Comment by sdisaacson** [comment]
OH God....My project was setup incorrectly.
It's all there.

**Reply by ElishaKay** [reply]
Awesome

U da man Scott!
